dat/missions/dvaered/colonel-transport.lua

```diff
@@ -28,13 +28,17 @@ local reward = 0
 
 local source_system = nil
 
-local ffriendly, fhostile -- codespell:ignore ffriendly
+local ffriendly = nil -- codespell:ignore ffriendly
+local fhostile = nil
 
 local npc_name = _("Dvaered Colonel")
 local npc_portrait = nil
 local npc_image = nil
+
+local m = vn.newCharacter( npc_name, {image=npc_image} )
+
 function create()
-   local spbs = lmisn.getSpobAtDistance( nil, 4, 6, "Dvaered" )
+   local spbs = misn.getSpobAtDistance( nil, 2, 6, "Dvaered" )
```

Comment:
No, this should be `lmisn`. If you haven't imported the library, you should add `local lmisn = require "lmisn"` at the top of the file (where the other requires are).